controller/commentsController.js

```diff
@@ -271,3 +272,29 @@ exports.deleteComment = async (req, res, next) => {
     return next(error);
   }
 };
+exports.getSingleComment = async (req, res, next) => {
+  const { refId } = req.query;
+  const commentId = req.params.commentId;
+  const applicationId = req.headers.token; //this will be retrieved from decoded api token after full auth implementation
+  const query = { applicationId: applicationId, _id: commentId };
+  if (refId) query.refId = refId;
+  try {
+    await Comments.find(query)
+      .populate("replies")
```

Comment:
We currently have an endpoint for getting replies and for performance reasons there is no need for populating the replies in the response. A summary of replies i.e numOfReplies will suffice. Thanks but you may remove this check the swagger docs for what you need to return